android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/Utils.java

```diff
@@ -58,6 +58,8 @@
   private static final String PWSTRIPTIME_KEY = "pwstriptime";
   private static final RegionResolver REGION_RESOLVER = new RegionResolver();
 
+  private static boolean userHasBeenWarned = false;
```

Comment:
I'm not sure where the best place to put this would be. As of right now the Utils class is the only area that knows about this `Toast`.